backend/inspirehep/mailing/providers/sendgrid.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import structlog
+from flask import current_app
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from inspirehep.mailing.errors import SendGridMissingAPIToken
+
+LOGGER = structlog.getLogger()
+
+
+def send_email(sender, receiver, subject, content, cc=[]):
+    """Send an email via SendGrid with the provided arguments.
+
+    Args:
+        sender (str): email address of the sender.
+        receiver (str): email address of the receiver.
+        subject (str): email's subject.
+        content (str): email's body.
+        cc (list): list of email addresses to put in cc.
+
+    Return:
+        None
+    """
+    api_key = current_app.config.get('SENDGRID_API_KEY')
+    if not api_key:
+        raise SendGridMissingAPIToken('Cannot authenticate to SendGrid. SENDGRID_API_KEY key not found.')
+
+    message = Mail(
+        from_email=sender,
+        to_emails=receiver,
+        subject=subject,
+        html_content=content
+    )
+    if cc:
```

Comment:
useless line, as the `for` in the next line will exit immediately if `cc` is an empty iterable.

Reply:
Leftover from the previous default value `None`